scripts/tour.js

```diff
@@ -0,0 +1,333 @@
+    }
+  }, {
+    element: ".nav.filteritems",
+    title: "Add Value",
+    content: "As we clicked on the attribute name in filter lists, this list came up. \
+  You can change your filters in here.",
+    placement: "left",
+    onPrev: function(tour) {
+      $(".field-enable.sidebar-toggle:eq(1)").click();
+    },
+    onNext: function(tour) {
+      $(".field-enable.sidebar-toggle:eq(1)").click();
+    }
+  }, {
+    element: ".search-box input:eq(0)",
+    title: "Search Attributes",
+    content: 'You can also search within attributes using this box. \
+  Just type something and list of attributes will be filtered.',
+    placement: "left",
+    onNext: function(tour) {
+      searchInputAttr = searchInputAttr ? searchInputAttr : getSearchTerm(true, $("#navcontainer a:visible:not(.view-attr)"));
+      $(".search-box input:eq(0)").val(searchInputAttr).trigger('input').keyup();
+    },
+    onPrev: function(tour) {
+      $("#filter .filter-item:visible a:not(:contains(Clear),.filter-link-cancel)").click();
+    }
+  }, {
+    element: "#navcontainer .sidebar-nav",
+    title: "Filtered Attribute List",
+    content: "As you can see after adding the term, this list is filtered.",
+    placement: "left",
+    onNext: function(tour) {
+      searchChosenAttr = searchChosenAttr ? searchChosenAttr : getSearchChosenAttribute();
+      searchChosenAttr.click();
+      $(".search-box input:eq(0)").val("").trigger('input').keyup();
+    },
+    onPrev: function(tour) {
+      $(".search-box input:eq(0)").val("").trigger('input').keyup();
+    }
+  }, {
+    element: ".search-box input:eq(2)",
+    title: "Search Values",
+    content: "You can also search in values of attributes.",
+    placement: "left",
+    onPrev: function(tour) {
+      $(".field-enable.sidebar-toggle:eq(1)").click();
+      $(".search-box input:eq(0)").val(searchInputAttr).trigger('input').keyup();
+    },
+    onNext: function(tour) {
+      searchInputValue = searchInputValue ? searchInputValue : getSearchTerm(false, $('.filteritems .field-enable:visible label'));
+      $(".search-box input:eq(2)").val(searchInputValue).trigger('input').keyup();
+    }
+  }, {
+    element: ".nav.filteritems",
+    title: "Filtered Values",
+    content: "As you can see the values are filtered based on search box.",
+    placement: "left",
+    onPrev: function(tour) {
+      $(".search-box input:eq(2)").val("").trigger('input').keyup();
+    },
+    onNext: function(tour) {
+      $(".search-box input:eq(2)").val("").trigger('input').keyup();
+      $(".field-enable.sidebar-toggle:eq(1)").click();
+    }
+  }, {
+    element: '#resultstable .panel:eq(0)',
+    title: "Dataset Details",
+    content: "By clicking on title of each dataset, you can see its details.",
+    placement: "right",
+    onPrev: function(tour) {
+      searchChosenAttr.click();
+      $(".search-box input:eq(2)").val(searchInputValue).trigger('input').keyup();
```

Comment:
Aref, the suggestion here was to add new class names that have nothing to do with the styling or functionality of the UI application. Just things to use as the selectors for use with the site tour instead of referencing things that are used by the app code (and could therefore be changed in the future).